include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -485,17 +507,47 @@ class FEEvaluationBase
    * Return the determinant of the Jacobian from the unit to the real cell
    * times the quadrature weight.
    */
-  VectorizedArray<Number> JxW(const unsigned int q_point) const;
+  VectorizedArray<Number> JxW(const unsigned int q_index) const;
+
+  /**
+   * Fills the JxW values currently used into the given array.
+   */
+  void
+  fill_JxW_values(AlignedVector<VectorizedArray<Number> > &JxW_values) const;
```

Comment:
why not `get_JxW_values() const` and maybe even return `AlignedVector<VectorizedArray<Number> >` (i guess we have move constructors for that combination)?

Reply:
This is pre-existing, it just moved slightly. I would be hesitant to change it right now because this PR is already quite big. If you don't mind I would open an issue (for the 9.1 release).

Reply:
sure, I did not notice that it's not the new functionality.

Reply:
This is now #6312.